src/components/sdk/DeviceSelection/MicSelection/index.tsx

```diff
@@ -14,21 +13,15 @@ interface Props extends BaseSdkProps {
   notFoundMsg?: string;
   /** The label that will be shown for microphone selection, it defaults to `Microphone source`. */
   label?: string;
-  /** A boolean that determines whether or not to include additional sample audio input devices, such as "None", "440 Hz". Defaults to true. This will be deprecated in the next major version. */
-  appendSampleDevices?: boolean;
 }
 
 export const MicSelection: React.FC<Props> = ({
   notFoundMsg = 'No microphone devices found',
   label = 'Microphone source',
-  appendSampleDevices = true,
   ...rest
 }) => {
-  const audioInputConfig: DeviceConfig = {
-    additionalDevices: appendSampleDevices,
-  };
   const selectAudioInput = useSelectAudioInputDevice();
-  const { devices, selectedDevice } = useAudioInputs(audioInputConfig);
+  const { devices, selectedDevice } = useAudioInputs();
 
   return (
     <DeviceInput
```

Comment:
Will now the prop on the `DeviceInput` be not called `selectedDeviceId` anymore since we are removing the concept of `deviceId` and moving to use `Device` (`AudioInputDevice` | `VideoInputDevice`)?

Reply:
This will be updated in the next PR.